modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/search/spi/model/query/contributor/DDMFormInstanceRecordKeywordQueryContributor.java

```diff
@@ -29,16 +39,104 @@ public void contribute(
 		String keywords, BooleanQuery booleanQuery,
 		KeywordQueryContributorHelper keywordQueryContributorHelper) {
 
+		SearchContext searchContext =
+			keywordQueryContributorHelper.getSearchContext();
+
 		queryHelper.addSearchTerm(
-			booleanQuery, keywordQueryContributorHelper.getSearchContext(),
-			Field.USER_NAME, false);
+			booleanQuery, searchContext, Field.USER_NAME, false);
 
-		queryHelper.addSearchLocalizedTerm(
-			booleanQuery, keywordQueryContributorHelper.getSearchContext(),
-			"ddmContent", false);
+		_addSearchLocalizedTerm(booleanQuery, Field.TITLE, searchContext);
```

Comment:
@G-Sa why do we also need to call this method for `Field.TITLE`? (which seems to be the Form title, and not a form field)

Reply:
this is actually for the form field 'Title' and i've put it there as another option and to make the search more extensive. It's not strictly needed so i could take it away at least for the scenario we had to cause this change